### ISSUE TITLE
Time tracking overview panel false positives for birdhouses and crops

### ISSUE THREAD

**Issue by deathbeam** [runelite/runelite#4783]
False "Ready" positives for:
- "Empty" birdhouses
- "Done" crops with "Weeds"

![screenshot 2018-08-08 20 06 45](https://user-images.githubusercontent.com/5115805/43855741-de28a8ac-9b46-11e8-90ca-8ed74c59d89f.png)
![screenshot 2018-08-08 20 06 36](https://user-images.githubusercontent.com/5115805/43855742-de790856-9b46-11e8-9fbd-18b6c07ab0a5.png)
![screenshot 2018-08-08 20 06 28](https://user-images.githubusercontent.com/5115805/43855743-de95ce1e-9b46-11e8-96c7-2fdf43de3ac7.png)
![screenshot 2018-08-08 20 06 14](https://user-images.githubusercontent.com/5115805/43855744-deb97d3c-9b46-11e8-93be-17aafef16207.png)

**Comment by takuyakanbr** [comment]
This was intentional. It shows "Ready" as long as there is no patch that is still growing or bird house that is still catching (i.e. "Ready" in this case means that you're ready to make a [patch type] run / bird house run).

**Comment by Abextm** [comment]
If we change this we should also change it in the non overview tabs.

**Comment by deathbeam** [comment]
I see, well as it shows "Ready" for both empty patches and birdhouses and trees that are done growing, its really misleading.

**Comment by Abextm** [comment]
Well, they are ready/done because they will not be changing in the future. For farming I tried to keep the number of special cases to a minimum, so weeds are not treated differently than any other crop, with the exception of autoweed.

**Comment by deathbeam** [comment]
Yeah I sort of understand why is it like it is now. But it is problem for overview panel because having no difference between you having stuff ready to be harvested and empty spots ready to be planted. It would be fine if you would always use all farming spots in each panel, but that is rarely the case, so you can be just stuck with perma "Ready" state if you e.g do just mahogany trees but not regular trees.
